Content.Shared/Chemistry/Components/ReactiveContainerComponent.cs

```diff
@@ -0,0 +1,22 @@
+namespace Content.Shared.Chemistry.Components;
+
+/// <summary>
+///     Represents a container that also contains a solution.
+///     This means that reactive entities react when inserted into the container.
+/// </summary>
+[RegisterComponent]
+public sealed partial class ReactiveContainerComponent : Component
+{
+    /// <summary>
+    ///     The container that holds the solution.
+    /// </summary>
+    [DataField("container", required: true)]
+    [ViewVariables]
+    public string Container = default!;
```

Comment:
```suggestion
    [DataField(required: true)]
    public string Container = default!;
```
DataField includes ViewVariables and the name gets automatically converted from camelCase to PascalCase nowadays